image_tools/src/showimage.cpp

```diff
@@ -34,15 +35,23 @@ encoding2mat_type(const std::string & encoding)
 {
   if (encoding == "mono8") {
     return CV_8UC1;
-  } else if (encoding == "bgr8") {
+  }
+  else if (encoding == "bgr8") {
```

Comment:
Tip: uncrustify / ament_uncrustify support the `--reformat` flag. It will lint the code for you ;)